pool/src/lib.rs

```diff
@@ -460,11 +460,20 @@ impl LendingPoolTrait for LendingPool {
         };
```

Comment:
Can we replace this block:
```
        let amount_to_withdraw = if amount == i128::MAX {
            who_balance
        } else {
            amount
         };
```
with `let amount_to_withdraw = amount.min(who_balance)`?

Reply:
This will result in skipping checking whether the user has sufficient balance to withdraw funds